src/Serilog.Sinks.Email/Sinks/Email/SecureSocketOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿#if MAIL_KIT
```

Comment:
This file should attribute the original source in MailKit (including its license header or link to the MailKit license).